templates/base/src/pages/_app.js

```diff
@@ -5,7 +5,7 @@ import GraphQLClient from '../graphQL/ApolloClient'
 import GlobalContextProvider from '../stores/global/ContextProvider';
 
 import { ThemeProvider } from 'styled-components';
-import { theme } from "../themes/theme";
+import { theme } from '../styles/themes/theme';
```

Comment:
We should be moving the theme to root/packages since for a whole project theme would be common. No point keeping it with each app. Sorry I should have thought of this earlier.